langchain/src/callbacks/handlers/tracer.ts

```diff
@@ -101,7 +101,7 @@ export abstract class BaseTracer extends BaseCallbackHandler {
       name: llm.name,
       parent_run_id: parentRunId,
       start_time: Date.now(),
-      end_time: 0,
+      end_time: Date.now(),
```

Comment:
This was done to maintain same behavior as python (otherwise, an in-progress run would show up with a negative latency)

Reply:
We shouldn't do this, we should just change the schema to make end_time nullable. We shouldn't save wrong information to the db